unlock-app/src/components/creator/CreatorLocks.js

```diff
@@ -54,23 +54,36 @@ export class CreatorLocks extends React.Component {
             pending
           />
         )}
-        {lockFeed.map(lock => {
-          return <CreatorLock key={JSON.stringify(lock)} lock={lock} />
-        })}
+        {lockFeed.length > 0 &&
+          lockFeed.map(lock => {
+            return <CreatorLock key={JSON.stringify(lock)} lock={lock} />
+          })}
+        {lockFeed.length === 0 && !showDashboardForm && (
+          <Container>
+            <Image src="/static/images/illustrations/lock.svg" />
+            <DimMessage>
+              <h1>Create a lock to get started</h1>
+              <p>
+                If you have already created some locks, they should appear here
```

Comment:
👍

Reply:
I wonder if a cleaner approach would not be to create a dedicated component for this?

Reply:
or maybe use `DefaultError` ?

Reply:
Sorry there's a bit of noise in here, this is based on the lock sorting branch, so there are commits from that in here. I wasn't sure what the best way to create a dependent PR was...